recipes/libhugetlbfs/build.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${ARCH} == "64" ]]; then
+    export ARCH=x86_64
+fi
```

Comment:
Why is this needed? Maybe worth including a comment if this is something the `Makefile` is using

Reply:
because the default makefile tries to check if ARCH is a known value and changes what it builds for it.